ui/main/src/app/modules/cards/services/handlebars.service.ts

```diff
@@ -340,6 +341,16 @@ export class HandlebarsService {
             return buffer;
         });
     }
+
+    private registerTextToHtml() {
```

Comment:
As this helper handle only end of line and spaces, call it something like `keepSpacesAndEndOfLine` may be less misleading. At the first glance, I thought this helper was able to manage strong, italic and stuff like that.